### PR TITLE
Added a CHERI_SDK variable to the Makefile for building on jenkins

### PR DESCRIPTION
**Pull request by gameboo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,19 +6,51 @@ all: jsrun
...
-CC=~/sdk/bin/clang
-CFLAGS+=-msoft-float -mabi=sandbox -cheri-linker
+CHERI_SDK?=~/sdk
+
+VERSION?=x86
+
+.if $(VERSION) == cheri128 || $(VERSION) == cheri256
+CC=$(CHERI_SDK)/bin/cheri-unknown-freebsd-clang
+CFLAGS+=-msoft-float
+CFLAGS+=-mabi=sandbox -cheri-linker
```
Comment (davidchisnall) [inline Makefile]: cheri-linker is no longer required.

Comment (gameboo) [reply Makefile]: well I guess we can get rid of it then

File: Makefile [hunk 2]
```diff
@@ -6,19 +6,51 @@ all: jsrun
...
+.if $(VERSION) == cheri128
+CFLAGS+=-mllvm -cheri128
+.endif
+#CFLAGS+=-mllvm -cheri-no-global-bounds
+CFLAGS+=-DDUK_USE_PACKED_TVAL=1
+CFLAGS+=--sysroot=$(CHERI_SDK)/sysroot
+CFLAGS+=-I/usr/include/edit
+CFLAGS+=-g
+LDFLAGS+=-mabi=sandbox -cheri-linker
```
Comment (gameboo) [inline Makefile]: is -cheri-linker needed in the LDFLAGS or is it also no longer needed here ?
Moreover,
I forgot to add
LDFLAGS+=-Wl,--whole-archive -lstatcounters -Wl,--no-whole-archive
to that pull request, if that could be merged in as well for the mips / cheri128 / cheri256 cases ?

Comment (davidchisnall) [reply Makefile]: It should never be needed.  You can add -no-cheri-linker if you want the old behaviour (but then things won't work with the standard CheriBSD libraries)

Comment (gameboo) [reply Makefile]: (Also am I supposed to resubmit a pull request with the changes or are you merging the changes ?)

**Pull request merged**